docs: have all checks run completed (#127)
reviewpad/docs: docs/guides/built-ins/functions/haveAllChecksRunCompleted/index.mdx

import CodeBlock from "@theme/CodeBlock";
import Example from "!!raw-loader!./example.yml";

**Description**:

> Checks if all check runs have completed on a pull request.

Reviewer: ```suggestion
Checks if all check runs for a pull request have been completed.
```

If the `check_runs_to_ignore` parameter is provided the checks with a name included in the parameter will be ignored.

> If the `conclusion` parameter is provided, all check runs must complete with provided conclusion.

Reviewer: ```suggestion
If the `conclusion` parameter is provided, all check runs must be complete with the provided conclusion.
```

Reviewer: Please provide the list of possible values for the conclusion.

**Parameters**:

| Parameter                             | Type       | Description                                                            |
| :------------------------------------ | :--------- | :--------------------------------------------------------------------- |

> | `check_runs_to_ignore` (optional)     | `[]string` | The names of the check runs to exclude from the completed check        |

Reviewer: ```suggestion
| `check_runs_to_ignore` (optional)  | `[]string` | The names of the check runs to exclude from the completed check |
```

Please the extra spaces at the end. Please also update the total number of `-----` on the line above.

| `conclusion` (optional)               | `string`   | The final conclusion of the check                                      |

**Return value**:

| Type           | Description                                                                                 |
| :------------- | :------------------------------------------------------------------------------------------ |

> | `boolean`      | `true` if all check runs have completed, `false` otherwise.                                 |

Reviewer: The same here.

**Examples**:

#### All check runs have completed.
```yml
$haveAllChecksRunCompleted()
```

> #### All check runs have completed except for integration-tests check.

Reviewer: ```suggestion
#### All check runs have been completed except for the `integration-tests` check.
```

```yml
$haveAllChecksRunCompleted(["integration-tests"])
```

> #### All check runs have completed successfully.

Reviewer: ```suggestion
#### All check runs have been completed with `success`.
```
